src/runtime.spec.ts

```diff
@@ -52,8 +53,27 @@ describe('Runtime', function test() {
     expect(runtime).to.be.ok;
   });
 
+  it('should switch the runtime for identity', async () => {
+    const runtime = await createDefaultRuntime(web3, dfs, runtimeConfig);
+    await runtime.profile.loadForAccount(runtime.profile.treeLabels.addressBook);
+    await runtime.profile.getIdentityAccessList();
+    await runtime.profile.setIdentityAccess(
+      identities[1],
+      dataKeys[web3.utils.sha3(identities[1])],
+    );
+
+    await runtime.profile.storeForAccount(runtime.profile.treeLabels.addressBook);
+    await runtime.profile.loadForAccount(runtime.profile.treeLabels.addressBook);
+    await runtime.profile.getIdentityAccessList();
+
+    const switchedRuntime = await getRuntimeForIdentity(runtime, identities[1]);
+    expect(runtime).to.be.ok;
+    expect(switchedRuntime).to.be.ok;
+    expect(switchedRuntime.profile).to.exist;
```

Comment:
Could we add some checks, if the correct identity was used? Currently also the current runtime could just be returned.